marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/EnvironmentController.java

```diff
@@ -86,12 +92,14 @@ public SystemInfo getSystemInfo(HttpSession session) {
         info.dataHubVersion = versionInfo.getHubVersion();
         info.marklogicVersion = versionInfo.getMarkLogicVersion();
         info.host = hubCentral.getHost();
-        info.pendoKey = "dummy-key-f7d836d7-2afa-40a6-44e4-f32f58cb35a";
         info.sessionTimeout = environment.getProperty("server.servlet.session.timeout");
         Object hubCentralSessionToken = session.getAttribute("hubCentralSessionToken");
         if (hubCentralSessionToken != null) {
             info.sessionToken = hubCentralSessionToken.toString();
         }
+        if (pendoKeyProvider != null) {
+            info.pendoKey = pendoKeyProvider.getPendoKey(PENDO_API_KEY);
```

Comment:
I don't think an argument should be needed here - we want EC to be as "dumb" as possible; it shouldn't know what the param name is or even that one is required. So the PKP interface should just be "getPendoKey()".

Reply:
Updated PR